index.org

```diff
@@ -843,14 +843,14 @@ helper-functions:
   (let [obj (js->clj o :keywordize-keys true)]
     (if (map? obj)
       (change-keys obj ->kebab-case)
-      (if (seq? obj)
+      (if (vector? obj)
```

Comment:
If we for sure it is always vector, i guess this is okay. Otherwise maybe `(seq obj)` (and check it is not `str`).

Reply:
At this stage the "obj" variable may be an object or a vector. I changed the seq? for vector? as the literal translation from a JS array object to clojure is in form of a vector

Reply:
Okay then `vector?` should be fine